lol-frontend/src/config/store/player/actions.ts

```diff
@@ -0,0 +1,5 @@
+export const editPlayerPosition = (position: Array<number>) =>
+  ({
+    type: "EDIT_PLAYER_POSITION",
+    position
+  } as const);
```

Comment:
For the actions you were pretty close each one should have a return type as well. So for what you were trying to do you would need to have defined your action types in a types or model file. 
model:
interface SetPlayerPositionAction {
    type: typeof SET_PLAYER_POSITION;
    payload: number[];
}
We need this because it will tell typescript exactly the shape our action should take.
Then in the action we should have something like this:
export function setPosition(newPosition: number[]): PlayerActionTypes {
    return {
        type: SET_PLAYER_POSITION,
        payload: newPosition
    }
}
(PlayerActionType is what you export from your type/model file)